chef_master/source/about_chefdk.rst

```diff
@@ -67,6 +67,8 @@ The most important tools included in the Chef development kit are:
      - Use to encrypt data bag items using the public keys of a list of nodes. This allows only those nodes to decrypt the encrypted values.
    * - ChefSpec
      - A unit testing framework that tests resources locally.
+   * - Cookstyle
+     - A Rubocop-based style-checking tool for writing clean cookbooks.
```

Comment:
For newcomers to Ruby this really doesn't provide anything, but for those that have done even a basic level of Ruby development I think this provides a bit of insight into what the tool is. When I explain to people that this is our replacement for Rubocop in cookbooks they get it. Otherwise they generally think it's a replacement for Foodcritic (which it should be).